app/Models/Foundation/Summit/Registration/SummitOrder.php

```diff
@@ -455,35 +455,35 @@ public function setOwnerEmail(string $owner_email): void
     /**
      * @return string
      */
-    public function getOwnerCompany(): ?string
+    public function getOwnerCompanyName(): ?string
     {
-        if($this->hasCompany())
-            return $this->company->getName();
-        return $this->owner_company;
+        if($this->hasCompany() && !is_null($this->owner_company))
```

Comment:
check for null is redundant
and we need to refactor hasCompany to hasCompanyOwner

Reply:
done